snap-core/src/main/java/org/esa/snap/core/dataio/geocoding/inverse/PixelQuadTreeInverse.java

```diff
@@ -59,7 +60,7 @@
     PixelQuadTreeInverse(boolean fractionalAccuracy) {
```

Comment:
Provide another constructor which gets as second parameter the interpolator enum and this can be used in tests.
The property does not need to be set and can't interferr with other tests.

Reply:
Done